src/style.css

```
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars > :not(:first-child) {
+  margin-left: 4px;
+}
```

```suggestion
.stars__star:not(:first-child) {
  margin-left: 4px;
}
```
redundant to make this styles dependent on parent selector